src/EditorFeatures/VisualBasicTest/GenerateConstructorFromMembers/GenerateConstructorFromMembersTests.vb

```diff
@@ -46,6 +46,33 @@ End Class",
 End Class")
         End Function
 
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateConstructorFromMembers)>
+        Public Async Function TestMultipleFields_VerticalSelection() As Task
+            Await TestInRegularAndScriptAsync(
+"Class Program[|
+    Private i As Integer
+    Private b As String|]
+End Class",
+"Class Program
+    Private i As Integer
+    Private b As String
+
+    Public Sub New(i As Integer, b As String{|Navigation:)|}
+        Me.i = i
+        Me.b = b
+    End Sub
+End Class")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateConstructorFromMembers)>
+        Public Async Function TestMultipleFields_SelectionIncludingClassName() As Task
+            Await TestMissingAsync(
+"Class Progra[|m
+    Private i As Integer
+    Private b As String|]
+End Class")
+        End Function
```

Comment:
can we have a test where the class has an "inherits" clause on the next line.  also one where the class is generic.  also one where there is a preceding function and a preceding field.

also test if we exactly select the fields and no whitespace. i.e.:

```vb
+    [|Private i As Integer
+    Private b As String|]
```